daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SValue.scala

```diff
@@ -179,8 +181,8 @@ object SValue {
     def id: Identifier
     def fields: ImmArray[Name]
     def values: util.ArrayList[SValue]
-    def lookupField(field: Name): SValue
-    def updateField(field: Name, value: SValue): SRecord
+    def lookupField(fieldNum: Int, field: Name): SValue
+    def updateField(fieldNum: Int, field: Name, value: SValue): SRecord
```

Comment:
We could go further. And remove passing the unused `field: Name`

Reply:
I am fine to let it like that. Maybe we can add a sanity check in the implementation or a least a comment.